packages/notification-service/test/exchange.test.ts

```diff
@@ -0,0 +1,26 @@
+import { CURRENCY_ENUM } from '@celo/utils'
+import Web3 from 'web3'
+import { getExchangeRate, getWeb3Instance } from '../src/exchange/exchangeQuery'
+
+describe('getExchangeRate', () => {
+  let web3: Web3
+  let dollarMakerRate: number
+  let goldMakerRate: number
+  it('should fetch a working web3 instance', async () => {
+    web3 = await getWeb3Instance()
```

Comment:
Are you mocking web3 for this? if not, I recommend we do.
If you need a semi-realistic web3 mock, you can use the one in `mobile/__mocks__`